server/config/settings.py

```diff
@@ -94,22 +94,26 @@
 SESSION_EXPIRE_AT_BROWSER_CLOSE = True
 
 LOGGING = {
-		'version': 1,
-		'disable_existing_loggers': False,
-		'handlers': {
-				'file': {
-						'level': 'DEBUG',
-						'class': 'logging.FileHandler',
-						'filename': 'logs/logs.txt',
-				},
+	'version': 1,
+	'disable_existing_loggers': False,
+	'handlers': {
+		'file': {
+			'level': 'WARNING',
+			'class': 'logging.FileHandler',
+			'filename': 'logs/logs.txt',
 		},
-		'loggers': {
-				'django': {
-						'handlers': ['file'],
-						'level': 'DEBUG',
-						'propagate': True,
-				},
+		'console': {
+		'level': 'INFO',
+		'class': 'logging.StreamHandler',
+	}
+	},
+	'loggers': {
+		'django': {
+			'handlers': ['file'],
+			'level': 'DEBUG',
+			'propagate': True,
 		},
```

Comment:
and error log for console?

Reply:
If I setup INFO level, it logs INFO and all after it (ERROR, WARNING, CRITICAL)

Reply:
ok, got it